### ISSUE TITLE
Allow user to choose font size

### ISSUE THREAD

**Issue by tspeed90** [tspeed90/tspeed90.github.io#12]
The page should have a few font size options displayed at the top to allow users to increase/decrease font size easily to meet their needs
